.gitignore

```diff
@@ -16,3 +16,4 @@ target
 # Ignore node modules
 node_modules/
 node/
+package-lock.json
```

Comment:
I think this is generally supposed to be checked in, no?

Reply:
`package-lock.json` gets generated every time I compile the project (through the `npm install` command), so I thought it would be better to not include it. `package.json` is checked in for sure!